app/assets/stylesheets/_utility.scss

```diff
@@ -0,0 +1,48 @@
+
+.m-b-30 {
+  margin-bottom: 30px;
+}
+
+.m-b-20 {
+  margin-bottom: 20px;
+}
+
+.m-b-10 {
+  margin-bottom: 10px;
+}
+
+.m-t-30 {
+  margin-top: 30px;
+}
+
+.m-t-20 {
+  margin-top: 20px;
+}
+
+.m-t-10 {
+  margin-top: 10px;
+}
+
+.m-l-30 {
+  margin-left: 30px;
+}
+
+.m-l-20 {
+  margin-left: 20px;
+}
+
+.m-l-10 {
+  margin-left: 10px;
+}
+
+.m-r-30 {
+  margin-right: 30px;
+}
+
+.m-r-20 {
+  margin-right: 20px;
+}
+
+.m-r-10 {
+  margin-right: 10px;
+}
```

Comment:
Not sure how I feel about this, especially since you're only using `.m-b-10` ;-)

Reply:
I've hit in to several occations throughout the project where that should have been, or can be used. And keep in mind that Bootstrap 4 actually has such a thing included. So it seems to make sense.